src/pages/WorldDetail/WorldDetail.tsx

```diff
@@ -24,6 +26,7 @@ export const WorldDetail = () => {
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const [courseData, setCourseData] = useState<any[]>();
   const [literatureOfVideo, setliteratureOfVideo] = useState<string>();
+  const [videoHasHomework, setHomework] = useState<boolean>();
```

Comment:
I would suggest to keep the naming of the getter and setter the same, so for example:

 [videoHasHomework, setVideoHasHomework]
 or
  [hasHomework, setHasHomework]